samples/matter/lock/sample.yaml

```diff
@@ -60,3 +60,25 @@ tests:
     integration_platforms:
       - nrf52840dk_nrf52840
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+  sample.matter.lock.release.ffs:
```

Comment:
Add all new configuration to quarantine so they will not build in integration jobs, that is move them above the comment line (line 35 here), and put them to scripts/quarantine.yml file.